chat/irc.properties

```diff
@@ -80,24 +78,24 @@ message.whoisEntry=\ua0\ua0\ua0\ua0%1$S: %2$S
 message.unknownNick=„%S” nie jest znanym pseudonimem użytkownika.
 message.channelKeyAdded=Użytkownik %1$S zmienił hasło dla kanału na „%2$S”.
 message.channelKeyRemoved=Użytkownik %S usunął hasło kanału.
-message.banMasks=Użytkownicy łączący się z następujących adresów są niedopuszczani do „%S”:
-message.noBanMasks=Brak wykluczonych adresów dla „%S”.
-message.banMaskAdded=Użytkownicy łączący się z adresów pasujących do „%1$S” zostali wykluczeni przez %2$S.
-message.banMaskRemoved=Użytkownicy łączący się z adresów pasujących do „%1$S” nie będą dłużej niedopuszczani przez %2$S.
+message.banMasks=Użytkownicy łączący się z następujących adresów są zbanowani z „%S”:
```

Comment:
It is jargon, blind calque and PWN makes me laugh limiting this to chat only.

Single translation would be better but I could live with inconsistent usage of existing Polish words expressing it correctly and I also like "wykluczać" most.